src/Service/ConfigService.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * Created by netlogix GmbH & Co. KG
+ *
+ * @copyright netlogix GmbH & Co. KG
+ */
+
+namespace Netlogix\NlxSwImgproxy\Service;
+
+use Netlogix\NlxSwImgproxy\Enum\ResizeType;
+use Shopware\Core\System\SystemConfig\SystemConfigService;
+
+/**
+ * @codeCoverageIgnore
+ */
+class ConfigService
+{
+    public const string CONFIG_DOMAIN = 'NlxSwImgproxy.config';
+
+    public function __construct(
+        private readonly SystemConfigService $systemConfigService
+    ) {
+    }
+
+    public function isEnabled(?string $salesChannelId = null): bool
+    {
+        return $this->systemConfigService->getBool($this->getConfigKey('enable'), $salesChannelId)
+            && $this->getBaseUrl($salesChannelId);
```

Comment:
Muss man hier explizit prüfen, ob `$this->getBaseUrl($salesChannelId) !== null `ist?